boards/arm/pinetime_devkit0/pinetime_devkit0_defconfig

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52832_QFAA=y
+CONFIG_BOARD_PINETIME_DEVKIT0=y
+CONFIG_HWINFO=y
+CONFIG_HWINFO_NRF=y
+
+# enable MPU
+CONFIG_ARM_MPU=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable UART driver
+CONFIG_SERIAL=y
+
+# enable I2C0 driver
+CONFIG_I2C=y
+CONFIG_I2C_0=y
+CONFIG_NRFX_TWIM=y
+CONFIG_NRFX_TWIM0=y
+
+# enable SPI1 driver
+CONFIG_NRFX_SPI=y
+CONFIG_NRFX_SPI1=y
```

Comment:
I guess it was supposed to be `CONFIG_SPI=y` instead. Or is the nrfx SPI driver indeed intended to be used directly (with [this](https://github.com/zephyrproject-rtos/hal_nordic/blob/master/nrfx/drivers/include/nrfx_spi.h) API), not through the spi_nrfx_spi shim and [Zephyr SPI API](https://github.com/zephyrproject-rtos/zephyr/blob/master/include/drivers/spi.h)?

Reply:
fixed it.